lib/modules/dosomething/dosomething_signup/includes/SignupTransformer.php

```diff
@@ -85,7 +85,11 @@ protected function transform($item) {
       $campaign = (object) $item->campaign;
       $current_run = $campaign->campaign_runs['current']['en']['id'];
       $current = ($item->campaign_run == $current_run);
-      $data += $this->transformSignup($item, $current);
+      $campaign_run = node_load($current_run);
+      $current_run_start = $campaign_run->field_run_date['en'][0]['value'];
+      $current_run_end = $campaign_run->field_run_date['en'][0]['value2'];
```

Comment:
@chloealee -- It's possible the Run may not have an end date set -- which would cause this to throw some exceptions. Can you confirm?